File: pinyougou-parent/pinyougou-sellergoods-interface/src/main/java/com/pinyougou/sellergoods/service/TypeTemplateService.java
package com.pinyougou.sellergoods.service;

import com.pinyougou.pojo.TbTypeTemplate;
import com.pinyougou.result.ResultPage;

public interface TypeTemplateService {
    ResultPage<TbTypeTemplate> showPageList(TbTypeTemplate typeTemplate,int page,int size);

    TbTypeTemplate findByid(Long id);

    void save(TbTypeTemplate typeTemplate);
}

File: pinyougou-parent/pinyougou-manager-web/src/main/webapp/js/service/typeTemplateService.js
app.service('typeTemplateService', function ($http) {
    this.showPageList = function (page, size, model) {
        return $http.post('/typeTemplate/showPageList.do?page=' + page + '&size=' + size, model);
    }

    this.findByid = function (id) {
        return $http.get('/typeTemplate/findByid.do?id=' + id);
    }

    this.save = function (model) {
        return $http.post('/typeTemplate/save.do?', model);
    }
})